src/scripts/main.js

```diff
@@ -2,4 +2,27 @@
 
 document.addEventListener('click', (e) => {
   // write code here
+  const spider = document.querySelector('.spider');
```

Comment:
This line of code will run every time a click event is triggered, which is not very efficient. The spider element doesn't change, so you can query it once outside of the event listener and reuse the reference. This will improve the performance of your code.